training/training.module

```diff
@@ -0,0 +1,378 @@
+    'page arguments' => array(3 ),
+    'access callback' => TRUE,
+    'file' => 'training.pages.inc',
+  );
+  $menu['training/menu/title'] = array(
+    'title callback' => 'training_menu_title_callback',
+    'access callback' => TRUE,
+    'file' => 'training.pages.inc',
+  );
+  $menu['training/menu/main'] = array(
+    'title' => 'Item in main menu',
+    'menu_name' => 'main-menu',
+    'access arguments' => array('train', ),
+    'file' => 'training.pages.inc',
+  );
+  $menu['training/theme'] = array(
+    'title' => 'Training Theme',
+    'description' => 'This page is Training Theme page',
+    'page callback' => 'training_theme_html',
+    'access callback' => TRUE,
+    'file' => 'training.pages.inc',
+    'expanded' => true,
+  );
+  $menu['training/theme/table'] = array(
+    'title' => 'Table Theme',
+    'description' => 'This page is Table Theme page',
+    'page callback' => 'training_theme_table',
+    'access callback' => TRUE,
+    'file' => 'training.pages.inc',
+  );
+  $menu['training/theme/image'] = array(
+    'title' => 'Image Theme',
+    'description' => 'This page is Image Theme page',
+    'page callback' => 'training_image_callback',
+    'access callback' => TRUE,
+    'file' => 'training.pages.inc',
+  );
+  $menu['training/theme/access'] = array(
+    'title' => 'Access Theme',
+    'description' => 'This page is Access Theme page',
+    'page callback' => 'training_access_callback',
+    'access callback' => TRUE,
+    'file' => 'training.pages.inc',
+  );
+  $menu['training/form'] = array(
+    'title' => 'Form Example',
+    'description' => 'This page is Form Example',
+    'page callback' => 'training_example_form_callback',
+    'page arguments' => array('training_google_form', ),
+    'access callback' => TRUE,
+    'file' => 'training.pages.inc',
+  );
+  return $menu;
+}
+
+/**
+ * Check access for page Custom Access Menu Item.
+ * $user - a global variable keeps the user basic information about the current user.
+ * $role_name - param contents role authenticated user
+ * @return bool
+ */
+function training_custom_access($role_name){
+  global $user;
+  return in_array($role_name, $user->roles);
+}
+
+/**
+ * Preprocessing argument. Returns the new value of the argument
+ * @param $arg - argument in URL training/menu/default_arg/%digit
+ * @return string
+ */
+
+function digit_load($arg){
+  switch ($arg) {
+    case 1:
+      $arg = 'one';
+      break;
+    case 2:
+      $arg = 'two';
+      break;
+    case 3:
+      $arg = 'three';
+      break;
+    case 99:
+      $arg = 'Almost there';
+      break;
+    default:
+      $arg = NULL;
+  }
+  return $arg;
+}
+
+/**
+ * Returns a dynamic page title
+ * string $user_name - contents user name
+ * @return string
+ */
+
+function training_menu_title_callback(){
+  global $user;
+  isset($user->name)?$user_name=$user->name:$user_name='anonymous';
+  return 'Dynamic title: username= '.$user_name;
+}
+
+
+/**
+ * Implements hook_permission()
```

Comment:
Put period (.) at the end of the function summary.
Remove parameters, because this is Drupal hook implementation
see https://www.drupal.org/coding-standards/docs